go/libkb/api_stub_html.go

```diff
@@ -0,0 +1,577 @@
+package libkb
```

Comment:
this file needs a comment header, and a build tag so this doesn't get into production builds (like in api_stub.go)

Reply:
corrected, wdyt now